src/models/ProfileModel.js

```diff
@@ -1,12 +1,22 @@
+import Immutable from 'immutable'
 import { abstractModel } from './AbstractModel'
 import validator from '../components/forms/validator'
 import ErrorList from '../components/forms/ErrorList'
 
 class ProfileModel extends abstractModel({
   name: null,
   email: null,
-  company: null
+  company: null,
+  tokens: new Immutable.Set(),
 }) {
+
+  constructor (data = {}) {
+    super({
+      ...data,
+      tokens: new Immutable.Set(data ? data.tokens : undefined)
```

Comment:
ok. Add todo in UserManagerDAO.getCBEList
to parse `const user = new ProfileModel(await this._ipfs(hash))` before create object and similar